bigtable/bttest/inmem.go

```diff
@@ -153,10 +153,11 @@ func (s *server) CreateTable(ctx context.Context, req *btapb.CreateTableRequest)
 	s.mu.Unlock()
 
 	ct := &btapb.Table{
-		Name:               tbl,
-		ColumnFamilies:     req.GetTable().GetColumnFamilies(),
-		Granularity:        req.GetTable().GetGranularity(),
-		DeletionProtection: req.GetTable().GetDeletionProtection(),
+		Name:                  tbl,
+		ColumnFamilies:        req.GetTable().GetColumnFamilies(),
+		Granularity:           req.GetTable().GetGranularity(),
+		DeletionProtection:    req.GetTable().GetDeletionProtection(),
+		AutomatedBackupConfig: req.GetTable().GetAutomatedBackupConfig(),
```

Comment:
Backups are not yet supported in emulator. So, backup policies should not be supported as well.
Does removing this show any errors ?

Reply:
Thanks 🙏🙏 I've removed it from the emulator, and disabled the test from running against the emulator (now only runs against Prod)